Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+        'Contents': r
+    }
+    if r['total_count'] != 0:
+        results = []
+        for issue in r['items']:
+            issue_details = makehash()
+            issue_details['ID'] = str(issue['id'])
+            issue_details['title'] = issue['title']
+            issue_details['link'] = issue['url']
+            issue_details['state'] = issue['state']
+            results.append(issue_details)
+            print_results['HumanReadable'] = tableToMarkdown(
+                str(r['total_count']) + ' found issues.\nComplete list of issues: ' + str(not r['incomplete_results']),
+                results, ['ID', 'title', 'state', 'link'])
+        print_results['EntryContext'] = {
+            'GitHub.Searches(val.query==obj.query)': {'query': query, 'results': results}
+        }
+    else:
+        print_results['HumanReadable'] = 'No issues found for this query!'
+    demisto.results(print_results)
+    return r
+
+
+def count_total_downloads():
+    r = http_request('GET', 'repos/teizenman/Demisto-start/releases')
+    output = []
+    for asset in r[0]['assets']:
+        count = makehash()
+        count['asset_name'] = asset['name']
+        count['download_count'] = asset['download_count']
+        output.append(count)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['markdown'],
+        'Contents': r,
+        'HumanReadable': tableToMarkdown('Download counts:', output, ['asset_name', 'download_count']),
+        'EntryContext': {
+            'GitHub.Download-counts(val.asset_name==obj.asset_name)': output
+        }
+    })
+    return r
+
+
+''' EXECUTION '''
+LOG('command is %s' % (demisto.command(),))
+try:
+    if demisto.command() == 'list-all-issues':
+        list_all_issues()
+
+    elif demisto.command() == 'create-issue':
+        title = demisto.args().get('title')
+        body = demisto.args().get('body')
+        assignees = argToList(demisto.args().get('assignees'))
+        labels = argToList(demisto.args().get('labels'))
+        create_issue(title, body, assignees, labels)
+
+    elif demisto.command() == 'edit-issue':
+        issue_num = demisto.args().get('issue_num')
+        title = demisto.args().get('title')
+        body = demisto.args().get('body')
+        assignees = argToList(demisto.args().get('assignees'))
+        labels = argToList(demisto.args().get('labels'))
+        edit_issue(issue_num, title, body, assignees, labels)
+
+    elif demisto.command() == 'close-issue':
+        issue_num = demisto.args().get('issue_num')
+        close_issue(issue_num, 'close')
+
+    elif demisto.command() == 'make-search':
+        search_query = demisto.args().get('search_query')
+        sort_by = demisto.args().get('sort_by')
+        order_by = demisto.args().get('order_by')
+        make_search(search_query, sort_by, order_by)
+
+    elif demisto.command() == 'count-downloads':
+        count_total_downloads()
+
+    elif demisto.command() == 'test-module':
```

Comment:
should be the first function in the elif.
We should only test the most basic functionality, the test-module is meant to check if the API key provided is correct or not, so no need to check every functions, and most certainly not functions that make changes to the service